### PR TITLE
Forms Automated Test - LPS-133783 | master

### PR DESCRIPTION
**Pull request by piresdm**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase [hunk 1]
```diff
@@ -367,14 +367,43 @@ definition {
...
+
+		FormFields.viewTextField(
+			fieldHelpText = "Test Help Text",
+			fieldLabel = "Text",
+			fieldName = "Text");
+
+		Form.viewFormPlaceholderText(key_value = "Test Placeholder Text");
+
+		FormFields.editText(
+			fieldName = "",
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase:401]: @piresdm, shouldn't we use the Text Field name here?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by natocesarrego** [comment]
@piresdm, can you please rebase your code against to the latest branch master of Brian Chan's repo to solve the reported conflicts?

Thanks.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase [hunk 2]
```diff
@@ -367,14 +367,43 @@ definition {
...
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		FormFields.setPlaceholder(fieldValue = "Test Placeholder Text");
```
[the remark this answers is not in the document]
Comment (piresdm) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase]: @natocesarrego done!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase (hunk 1 above)
Comment (piresdm) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase]: @natocesarrego It doesn't need, but I included it anyway!

**Comment by piresdm** [comment]
> @piresdm, can you please rebase your code against to the latest branch master of Brian Chan's repo to solve the reported conflicts?
> 
> Thanks.
@natocesarrego done!

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
@piresdm, please check the SF issue.

Thanks.

**Comment by wesleyroberts** [comment]
This test was sent in this pull request https://github.com/liferay-forms/liferay-portal/pull/674

**Pull request closed without merging**